bigbluebutton-html5/imports/api/2.0/users/server/eventHandlers.js

```diff
@@ -17,4 +18,5 @@ RedisPubSub.on('UserEmojiChangedEvtMsg', handleEmojiStatus);
 RedisPubSub.on('SyncGetUsersMeetingRespMsg', handleGetUsers);
 RedisPubSub.on('GuestsWaitingForApprovalEvtMsg', handleGuestsWaitingForApproval);
 RedisPubSub.on('GuestApprovedEvtMsg', handleGuestApproved);
+RedisPubSub.on('UserEjectedFromMeetingEvtMsg', handleUserEjected);
```

Comment:
There's one more event happening when you kick an user that is not being handled `DisconnectClientSysMsg`.
I'm not sure what are the purpose of this event, maybe add an SYSTEM_MESSAGE in the chat?

cc @antobinary

Reply:
*SysMsg are messages between server components. We do not need to handle this afaik. It is akka-apps -> red5 bigbluebutton-apps to close the connection to a flash client